packages/api-openfin/src/window.ts

```diff
@@ -129,7 +129,7 @@ const convertOptions = (options: ssf.WindowOptions) => {
 };
 
 class Window implements ssf.Window {
-  eventListeners: Map<any, any>;
+  eventListeners: Map<string, ((...args: any[]) => void)[]> = new Map();
```

Comment:
We also do `this.eventListeners = new Map();` in the constructor, which is now redundant.